### PR TITLE
Update recruitment and compensation sections

### PR DESCRIPTION
**Pull request by bpdesigns**
Updated content based to address [#126 Write section on ensuring diverse research participants](https://github.com/18F/ux-guide/issues/126). 

@jstrothman I've add a link to your Bias Identification workshop under "Writing a research plan > Team participation"

@MelissaBraxton I used your scratchpad in #126 as a starting point for these edits. 

In addition to a general review of the new text, I think the "Recruitment for Behavioral Diversity" might need to be added to or rewritten. I also think we could invite more participation in these edits, but thought I'd get your thoughts first.

### CODE REVIEW

**Comment by bpdesigns** [comment]
@megan-moser, @Mtcarp, @em-herrick, @thebestsophist for your awareness and input.

File: _pages/research/plan.md [hunk 1]
```diff
@@ -163,17 +171,41 @@ If you’re doing usability testing, consider the following questions as well:
...
 
 #### Why do we offer compensation?
 
-We compensate participants for more than just the time they spend speaking with us. There can be additional costs like transportation, time off from work, and child care. We also compensate to show we value their lived experiences and expertise. Sometimes we ask participants to imagine or recall a painful personal experience, including previous difficulties that resulted from interactions with government services.
+Many government services support people through stressful situations. We learn the usability limitations of our designs when we get input from people with lived experience and diverse perspectives. We compensate participants for more than just the time they spend speaking with us. There can be additional costs like transportation, time off from work, and child care. We also compensate to show we value their lived experiences and expertise. Sometimes we ask participants to imagine or recall a painful personal experience, including previous difficulties that resulted from interactions with government services.
```
Comment (MelissaBraxton) [inline _pages/research/plan.md:208]: I feel like there's a missed opportunity here to point how compensation helps us reduce bias in our research by not necessarily restricting participation to folks who have the privilege of being able to donate their time unpaid. I feel like that reason might be priority to stress over recompense for possibly asking folks to recall a stressful situation--esp. if we aren't balancing mention of this possible harm w/ content or at least linking to content on how we make sure in our research to reduce the risk of re-traumatizing participants.

**Review by MelissaBraxton — CHANGES_REQUESTED** [review]
Added some thoughts on refinements. 

I think it'd also be good to standardize or at least reduce the number of terms we use to talk about participants. It's a bit confusing that throughout this page, they're referred to as participant, interviewee, audience of interest, user of your service, target audience, target group, and participant group.

File: _pages/research/plan.md [hunk 2]
```diff
@@ -94,7 +94,11 @@ Choose one or more methods appropriate for meeting your goals and answering your
...
+
+Consider who’s perspective might be missing from the planning process. Any opportunities to include people who have direct experience using the product or service or people interacting regularly with those who do will help the team get closer to building with, not just for, the people who will use or be impacted by the outcomes of our research. 
+
+Running a [Bias Identification workshop(18F/GSA access only](https://docs.google.com/presentation/d/1l9UrizeRtiYrdclCRb7LIxPQF6XTvCI5A58ty-blRj4/edit#slide=id.ga8f8d32361_0_0)) helps teams account for their unconscious bias that might impact the research. The team should also collectively review the Bias and Ethics pages on the 18F UX Guide to see if their plan accounts for bias and ethics of the research they are about to engage in.
```
[the remark this answers is not in the document]
Comment (bpdesigns) [reply _pages/research/plan.md:101]: updated

File: _pages/research/plan.md [hunk 3]
```diff
@@ -132,13 +136,17 @@ Here’s a sample timeline for a contextual inquiry (on site) followed by eight
 
 ## Participants and recruiting
 
-Most of 18F’s design research depends on you directly interacting with people. Who those people are matters. Participants are the people you’ll recruit to take part in your research. For planning purposes, recruiting involves identifying target groups and defining your recruitment criteria relative to your research question.
+Most of 18F’s design research depends on you directly interacting with people. Who those people are matters. Participants are the people you’ll recruit to take part in your research. For planning purposes, recruiting involves identifying target groups and defining your recruitment criteria relative to your research question. At 18F we are often designing for the diverse U.S. population. It is our responsibility to include and learn from people with a range of perspectives and a diversity of needs. Our products and services need to be accessible to all users regardless of their abilities and we need to consider equity of access and experience as we determine who to include in our research.
```
[the remark this answers is not in the document]
Comment (bpdesigns) [reply _pages/research/plan.md:139]: Thoughts on this edit? "To make our products and services accessible to all, regardless of their abilities we need to take into account the degree of difficultly different groups might have when determining who to include in our research."

File: _pages/research/plan.md [hunk 4]
```diff
@@ -132,13 +136,17 @@ Here’s a sample timeline for a contextual inquiry (on site) followed by eight
...
+
+
+### Identifying participant groups
 
-### Identifying target groups
+Because of the time-limited nature of 18F engagements, participant groups can depend on the [type of research]({{site.baseurl}}/research/clarify-the-basics/#research-types) you're doing and where you’re at in the overall design process. For example, if you’re doing stakeholder interviews as part of a Path Analysis project, you’re likely to learn more about who you need to talk to with each interview you do. As a result, you can often rely on opportunistic tactics like “snowball sampling” (asking interviewees “who should I talk to next?”) instead of explicitly identified groups. If you start with snowball sampling, make it a priority in future rounds of research to learn from people within your audience of interest.
```
[the remark this answers is not in the document]
Comment (bpdesigns) [reply _pages/research/plan.md:144]: Updated as: "An opportunistic tactic like asking interviewees “who should I talk to next?” to get an initial understanding of what the relevant segments or audiences of interest. However, in later rounds of research make it a priority to learn from people within your audience of interest."

File: _pages/research/plan.md [hunk 5]
```diff
@@ -163,17 +171,41 @@ If you’re doing usability testing, consider the following questions as well:
...
+- 
+- **Intercept testing in government buildings.** Doing in-person research in public buildings that are visited by a wide cross-section of the population, such as libraries and post-offices, is one way to reach a diverse group of participants. If you reach a point where you realize you’ve excluded a specific set of users who will use your product/service, intercept testing can be a great save. Intercept testing can be used as a way to be intentional about testing with a specific set of diverse users. For example, setting up intercept testing at a library in a low income neighborhood might increase a team’s chances of ensuring some of the feedback on a product or service comes directly from low income users.
+
+#### Recruitment for Behavioral Diversity 
+People who look differently may behave similarly when using a product or service. Creating [Personas](https://methods.18f.gov/decide/personas/) allows you to define and distinguish between the way different groups of people perceive or behave towards your designs.
```
[the remark this answers is not in the document]
Comment (bpdesigns) [reply _pages/research/plan.md:186]: done

File: _pages/research/plan.md (hunk 1 above)
Comment (bpdesigns) [reply _pages/research/plan.md:208]: great point.

Comment (MelissaBraxton) [reply _pages/research/plan.md]: I think we need to revise this content before merging.

File: _pages/research/plan.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bpdesigns) [reply _pages/research/plan.md]: @MelissaBraxton does this work better?

File: _pages/research/plan.md (hunk 1 above)
Comment (bpdesigns) [reply _pages/research/plan.md]: ready for review @MelissaBraxton

File: _pages/research/plan.md [hunk 6]
```diff
@@ -163,17 +171,41 @@ If you’re doing usability testing, consider the following questions as well:
...
 
 #### Why do we offer compensation?
 
-We compensate participants for more than just the time they spend speaking with us. There can be additional costs like transportation, time off from work, and child care. We also compensate to show we value their lived experiences and expertise. Sometimes we ask participants to imagine or recall a painful personal experience, including previous difficulties that resulted from interactions with government services.
+Compensation for participation in research allows us to get perspectives from people who have the privilege and flexibility to donate their time unpaid. We learn the usability limitations of our designs when we get input from people with lived experience and diverse perspectives. We compensate participants for more than just the time they spend speaking with us. There can be additional costs like transportation, time off from work, and child care. We also compensate to show we value their lived experiences and expertise. Sometimes we ask participants to imagine or recall a painful personal experience, including previous difficulties that resulted from interactions with government services.
```
Comment (MelissaBraxton) [inline _pages/research/plan.md:208]: There may be a typo here, but as written it sounds like compensation is for people who _can_ donate their time? 

What about something like: "Compensating participants helps us reduce bias in our research. Not compensating research participants can limit our participant pool to people who have the privilege and flexibility to donate their time unpaid."

Comment (bpdesigns) [reply _pages/research/plan.md:208]: That sounds good to me. I was missing a critical "don't" in the earlier version.

Comment (bpdesigns) [reply _pages/research/plan.md]: @MelissaBraxton updated with your edit. With your review I think the content is ready to merge.

**Comment by bpdesigns** [comment]
Following the fail on verifying links I see "Error: in _site/research/lead/index.html" and  "Error: in _site/research/plan/index.html" 

I only updated the Plan file. @mgwalker can we cowork on troubleshooting this next week?

File: _pages/research/plan.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MelissaBraxton) [reply _pages/research/plan.md]: Yep!

**Review by MelissaBraxton — APPROVED** [review]

File: _pages/research/plan.md [hunk 7]
```diff
@@ -136,12 +136,12 @@ Here’s a sample timeline for a contextual inquiry (on site) followed by eight
...
+Because of the time-limited nature of 18F engagements, participant groups can depend on the [type of research]({{site.baseurl}}/research/clarify-the-basics/#research-types) you're doing and where you’re at in the overall design process. For example, if you’re doing stakeholder interviews as part of a Path Analysis project, you’re likely to learn more about who you need to talk to with each interview you do. An opportunistic tactic like asking interviewees “who should I talk to next?” to get an initial understanding of what the relevant segments or audiences of interest. However, in later rounds of research make it a priority to learn from people within your audience of interest.
```
Comment (MelissaBraxton) [inline _pages/research/plan.md:144]: There's a typo here. 

Suggested edit:
"An opportunistic tactic like asking interviewees “who should I talk to next?” can help you get an initial understanding of the different user groups who's needs you must design for. You may need to focus later rounds of research on learning directly from people within these user groups."

File: _pages/research/plan.md (hunk 6 above)
Comment (MelissaBraxton) [reply _pages/research/plan.md]: LGTM!

File: _pages/research/plan.md (hunk 7 above)
Comment (bpdesigns) [reply _pages/research/plan.md]: @MelissaBraxton updated and ready for your review.

**Comment by jstrothman** [comment]
I updated the frames of reference deck to be a stand-alone activity--it was a deck with some framing slides for a Research Guild discussion. As a result, I updated the link to the deck to go to the first slide instead of mid-way through. I also updated the frames of reference form (not directly linked in this page) to have some background info, now that it's been [linked to from CFPB](https://cfpb.github.io/design-system/guidelines/setting-the-foundation#frames-of-reference).

I'd like to do a bit more review later today, if that's okay.

**Comment by bpdesigns** [comment]
Sounds good. Thanks @jstrothman

**Review by jstrothman — COMMENTED** [review]
@bpdesigns The previous changes look good to me! I made a bunch of edits, so I'm not using the 'approve' feature in this review as you may want to revert some of the edits or make further adjustments

**Review by bpdesigns — COMMENTED** [review]
These look great. Thanks @jstrothman

**Comment by bpdesigns** [comment]
Thanks @jstrothman and @MelissaBraxton for reviewing all this text!

**Review by bpdesigns — COMMENTED** [review]
@MelissaBraxton can you approve this latest commit? Julie updated the text. It looks good to me but as the original author of the commit I can't approve it.

**Pull request merged**